frontend/src/hooks/useApi.ts

```diff
@@ -113,5 +116,12 @@ export const useApi = (): PimsAPI => {
     return data;
   };
 
+  axios.loadProperties = async (params?: IGeoSearchParams): Promise<any[]> => {
+    const { data } = await axios.get<any[]>(
```

Comment:
Please at least some basic error handling logic since this is a core application request that fails periodically.

Reply:
Also I think this will not display the loading bar anymore right? Refactoring is great but lets not regress functionality.